trade/trader_stakecube.js

```diff
@@ -468,33 +480,37 @@ module.exports = (
       const paramString = `pair: ${pair}`;
       const pair_ = formatPairName(pair);
 
-      let ticker;
+      let scTickerData;
 
       try {
-        ticker = await stakeCubeApiClient.ticker(pair_.pair);
+        scTickerData = await stakeCubeApiClient.ticker(pair_.pair);
 
-        if (ticker.errorMessage || !ticker[pair_.pair]) {
-          log.warn(`API request getRates-ticker(${paramString}) of ${utils.getModuleName(module.id)} module failed. ${ticker.errorMessage}`);
+        if (scTickerData.error || !scTickerData.result[pair_.pair]) {
+          log.warn(`API request getRates-ticker(${paramString}) of ${utils.getModuleName(module.id)} module failed. ${scTickerData.error}`);
           return undefined;
         }
       } catch (err) {
         log.warn(`API request getRates-ticker(${paramString}) of ${utils.getModuleName(module.id)} module failed. ${err}`);
         return undefined;
       }
 
-      let orderBook;
+      let ticker = scTickerData.result;
+
+      let scOrderBookData;
       try {
-        orderBook = await stakeCubeApiClient.orderBook(pair_.pair);
+        scOrderBookData = await stakeCubeApiClient.orderBook(pair_.pair);
 
-        if (orderBook.errorMessage) {
-          log.warn(`API request getRates-orderBook(${paramString}) of ${utils.getModuleName(module.id)} module failed. ${orderBook.errorMessage}`);
+        if (scOrderBookData.error) {
```

Comment:
It's not necessary to check this as in case of error, function will be try-catched.
In case of error, we already logged it in `stakecube_api`.